source/_docs/terminus/plugins/create.md

```diff
@@ -0,0 +1,157 @@
+Run the command with the verbose option if it does not work as expected:
+<div class="copy-snippet">
+<button class="btn btn-default btn-clippy" data-clipboard-target="#terminus-hello-vvv"><img class="clippy" src="/source/docs/assets/images/clippy.svg" width="17" alt="Copy to clipboard"></button>
+<figure><pre id="terminus-hello-vvv"><code class="bash" data-lang="bash">terminus hello -vvv</code></pre></figure>
+</div>
+If Terminus had trouble loading your plugin then there should be a debug notice logged. You may also be able to get more info by looking in your PHP error logs.
+
+## Distribute Plugin
+While the plugin you’ve created is good enough for local development, if you create something that you’re proud of, share it with the world! There are a few more things you’ll need to do to make your plugin distributable:
+
+### Vendor Name
+Add a vendor name to the plugin name within `composer.json` so that your plugin may be distinguished from other plugins that might share the same name. Most people use their GitHub user or organization name for the vendor. For a plugin distributed by Pantheon (GitHub organization: ‘pantheon-systems’) we would change the name field to:
+```
+{
+    "name": "pantheon-systems\terminus-hello-world",
+}
+```
+### PSR-4 Namespacing
+Your plugin command class name may eventually conflict with internal or third-party commands. To avoid this you will need to add a PSR-4 compatible namespace. This should contain your vendor name and the plugin name. Add a `namespace` declaration to the top of your php file (e.g. `$HOME/.terminus/plugins/hello-world/src/HelloCommand.php`):
+<div class="copy-snippet">
+<button class="btn btn-default btn-clippy" data-clipboard-target="#terminus-plugin-namespace"><img class="clippy" src="/source/docs/assets/images/clippy.svg" width="17" alt="Copy to clipboard"></button>
+<figure><pre id="terminus-plugin-namespace"><code class="bash" data-lang="bash">namespace Pantheon\TerminusHelloWorld\Commands;</code></pre></figure>
+</div>
+The `Commands` part of the namespace is not strictly necessary but it can help keep things organized if you need to add supporting classes to your plugin.
+
+You will also need to make sure your src directory and composer file reflect the new namespace. Move the `HelloCommand.php` file  from `src/` to the `src/Commands` directory to mirror the last part of the namespace. If you have a lot of commands in your plugin, you can group them into command groups by adding another layer to the namespace and directory structure.
+
+Update the `composer.json` file with an autoload section to indicate how to load your namespace. Your composer file should now look like:
+```
+{
+    "name": "pantheon-systems\terminus-hello-world",
+    "description": "An Hello, World Terminus command",
+    "type": "terminus-plugin",
+    "autoload": {
+        "psr-4": { "Pantheon\\TerminusHello": "src" }
+    },
+    "extra": {
+        "terminus": {
+            "compatible-version": "1.*"
```

Comment:
I recommend `"^1"` here instead of `"1.*"`, but @ronan may still disagree with this. Ensure uniformity with the Terminus Plugin Example.